tools/format_python_tools.sh

```diff
@@ -15,11 +15,15 @@ echo "Running Python format check..."
 python format_python_tools.py $1
 
 echo "Running Python3 flake8 check..."
-EXCLUDE="--exclude=../benchmarks/tmp/*,*/venv/*"
+cd ..
+EXCLUDE="--exclude=benchmarks/tmp/*,*/venv/*,tools/format_python_tools.py,tools/gen_compilation_database.py,bazel-*"
 flake8 . ${EXCLUDE} --count --select=E901,E999,F821,F822,F823 --show-source --statistics
 # We raise the bar higher for benchmarks/ overall, but especially when it comes to docstrings.
 # Check everything, except indentation and line length for now.
-# Also, we ignore unused imports and redefinitions of unused, as those seems to raise false flags in test definitions.
-flake8 ../benchmarks/ ${EXCLUDE} --docstring-convention pep257 --ignore=E114,E111,E501,F401,F811 --count --show-source --statistics
+# We ignore unused imports and redefinitions of unused, as those seems to raise false flags in test definitions.
+# We ignore formatting isssues E124/E125/E126 because they conflict with the automtic fix format feature.
```

Comment:
Are we using the wrong auto formatter? Which formatting issues are we ignoring here?